step_080_implement_a_delay_element/src/main/scala/org/applied/akkastreams/FilterElements.scala

```diff
@@ -29,9 +29,9 @@ object FilterElements {
         val eq = MQueue(List.fill(delay)(0.0d): _*)
 
         { case (sample, ff) =>
-          eq.enqueue(sample)
-          val delayedSample = eq.dequeue()
-          Iterable((delayedSample, ff + delayedSample * scaleFactor))
+            eq.enqueue(sample)
```

Comment:
This reminds me that it would be nice to have a formatter defined in the project such as Scalafmt

Reply:
I agree about the usefulness to add scalafmt.

One thing I'm clueless about is to find the scalafmt config setting that formats the following snippet as shown:

```
object DelayLineFlow {
    def apply(delay: Int, scaleFactor: Double): Flow[(Double, Double), (Double, Double), NotUsed] = {
      Flow[(Double, Double)].statefulMapConcat { () =>
        // mutable state needs to be kept inside the stage
        val eq = Array.fill(delay)(0.0d)
        var idx = 0

        { case (sample, ff) =>
            val delayedSample = eq(idx)
            eq(idx) = sample
            idx = (idx + 1) % delay
            Iterable((delayedSample, ff + delayedSample * scaleFactor))
        }
      }
    }
  }
```

Played with a number of settings, but haven't found a way to avoid it being formatted by scalafmt as follows:

```
  object DelayLineFlow {
    def apply(delay: Int, scaleFactor: Double): Flow[(Double, Double), (Double, Double), NotUsed] = {
      Flow[(Double, Double)].statefulMapConcat { () =>
        // mutable state needs to be kept inside the stage
        val eq = Array.fill(delay)(0.0d)
        var idx = 0

        {
          case (sample, ff) =>
            val delayedSample = eq(idx)
            eq(idx) = sample
            idx = (idx + 1) % delay
            Iterable((delayedSample, ff + delayedSample * scaleFactor))
        }
      }
    }
  }
```

The `case ...` should be in the same line as the opening curly brace.
One of the reasons is that in Scala 3, this wouldn't compile

Reply:
Going to merge as-is for the time being. I'll create an issue to remember that this needs to be looked at.